common/cache/domainCache.go

```diff
@@ -131,6 +131,7 @@ type (
 		failoverVersion             int64
 		isGlobalDomain              bool
 		failoverNotificationVersion int64
+		failoverEndTime             *int64
```

Comment:
do we need 3 states for failoverEndTime? or make it int64 is enough?

Reply:
Yes. I don't want to use a special value to indicate the end time is not set. Just use a pointer here.